workflow/routes/routes.go

```diff
@@ -0,0 +1,12 @@
+package Routes
+
+import (
+	"github.com/E-Furqan/Food-Delivery-System/Controllers/orderControllers"
+	"github.com/gin-gonic/gin"
+)
+
+func Order_routes(orderController orderControllers.OrderControllerInterface, server *gin.Engine) {
```

Comment:
again these should be in order service

Reply:
like we need to create a folder named service and move it there or we need to just move them in orderclient and for data pipeline in data pipeline client